pandas/tseries/resample.py

```diff
@@ -14,7 +21,560 @@
 import pandas.tslib as tslib
 
 
-_DEFAULT_METHOD = 'mean'
+class Resampler(_GroupBy):
+
+    # to the groupby descriptor
+    _attributes = ['freq', 'axis', 'closed', 'label', 'convention',
+                   'loffset', 'base', 'kind']
+
+    # API compat of allowed attributes
+    _deprecated_valids = _attributes + ['_ipython_display_', '__doc__',
+                                        '_cache', '_attributes', 'binner',
+                                        'grouper', 'groupby', 'keys',
+                                        'sort', 'kind', 'squeeze',
+                                        'group_keys', 'as_index',
+                                        'exclusions']
+
+    # API compat of disallowed attributes
+    _deprecated_invalids = ['iloc', 'loc', 'ix']
```

Comment:
yep, no problem

I only would then make it a bit more clear in the whatsnew that it is not anly assignment that will raise an error, but also getting values